repo/open.go

```diff
@@ -362,8 +370,8 @@ func readFormatBlobBytesFromCache(ctx context.Context, cachedFile string, validD
 	return os.ReadFile(cachedFile) //nolint:wrapcheck,gosec
 }
 
-func readAndCacheFormatBlobBytes(ctx context.Context, st blob.Storage, cacheDirectory string, validDuration time.Duration) ([]byte, error) {
-	cachedFile := filepath.Join(cacheDirectory, "kopia.repository")
+func readAndCacheRepositoryBlobBytes(ctx context.Context, st blob.Storage, cacheDirectory, blobID string, validDuration time.Duration) ([]byte, error) {
+	cachedFile := filepath.Join(cacheDirectory, blobID)
 
 	if validDuration == 0 {
 		validDuration = defaultFormatBlobCacheDuration
```

Comment:
maybe this default value should be renamed too? Applies to more than just format blobs

Reply:
Yes. Will change.